docs/src/getting_started.md

```diff
@@ -1,41 +1,39 @@
-# Getting Started with Optimization in Julia
+# Getting Started with Optimization.jl
 
 In this tutorial, we introduce the basics of Optimization.jl by showing
-how to easily mix local optimizers from Optim.jl and global optimizers
-from BlackBoxOptim.jl on the Rosenbrock equation. The simplest copy-pasteable
-code to get started is the following:
+how to easily mix local optimizers and global optimizers on the Rosenbrock equation. 
+The simplest copy-pasteable code using a quasi-Newton method (LBFGS) to solve the Rosenbrock problem is the following:
 
 ```@example intro
 # Import the package and define the problem to optimize
-using Optimization
+using Optimization, Zygote
 rosenbrock(u, p) = (p[1] - u[1])^2 + p[2] * (u[2] - u[1]^2)^2
 u0 = zeros(2)
 p = [1.0, 100.0]
 
-prob = OptimizationProblem(rosenbrock, u0, p)
-
-# Import a solver package and solve the optimization problem
-using OptimizationOptimJL
-sol = solve(prob, NelderMead())
+optf = OptimizationFunction(rosenbrock, AutoZygote())
+prob = OptimizationProblem(optf, u0, p)
 
-# Import a different solver package and solve the optimization problem a different way
-using OptimizationBBO
-prob = OptimizationProblem(rosenbrock, u0, p, lb = [-1.0, -1.0], ub = [1.0, 1.0])
-sol = solve(prob, BBO_adaptive_de_rand_1_bin_radiuslimited())
+sol = solve(prob, Optimization.LBFGS())
 ```
 
-Notice that Optimization.jl is the core glue package that holds all the common
-pieces, but to solve the equations, we need to use a solver package. Here, OptimizationOptimJL
-is for [Optim.jl](https://github.com/JuliaNLSolvers/Optim.jl) and OptimizationBBO is for
-[BlackBoxOptim.jl](https://github.com/robertfeldt/BlackBoxOptim.jl).
+## Import a different solver package and solve the problem
+
+OptimizationOptimJL is a wrapper for [Optim.jl](https://github.com/JuliaNLSolvers/Optim.jl) and OptimizationBBO is a wrapper for [BlackBoxOptim.jl](https://github.com/robertfeldt/BlackBoxOptim.jl).
 
-The output of the first optimization task (with the `NelderMead()` algorithm)
-is given below:
+First let's use the NelderMead a derivative free solver from Optim.jl:
 
 ```@example intro
-optf = OptimizationFunction(rosenbrock, Optimization.AutoForwardDiff())
-prob = OptimizationProblem(optf, u0, p, lb = [-1.0, -1.0], ub = [1.0, 1.0])
-sol = solve(prob, NelderMead())
+using OptimizationOptimJL
+sol = solve(prob, Optim.NelderMead())
```

Comment:
We probably shouldn't lead people to `NelderMead` at all.

Reply:
For a derivative-free local method it's decent and Optim is quite popular so it makes sense to have it for now, PRIMA has been working pretty well for me on some problems and we might want to start highlighting that more but I am still not as sure about it